### ISSUE TITLE
Override core TOC nesting

### ISSUE THREAD

**Issue by farling42** [farling42/fvtt-pdf-pager#63]
Some PDFs allow more than 2 levels of Outline, so provide all of these in the Journal's TOC.

**Comment by farling42** [comment]
See _renderHeadings in foundry.js where there is a hard-coded test for `if ( level < minLevel + 2 )`

**Comment by farling42** [comment]
D:\FoundryVTT\Node\11\resources\app\public\scripts\foundry.js line 73973

D:\FoundryVTT\Node\11\resources\app\public\css\style.css: line 5309, add
```
.sheet.journal-entry .pages-list .h4 .heading-link {
  text-indent: calc(3 * var(--level-indent));
}
.sheet.journal-entry .pages-list .h5 .heading-link {
  text-indent: calc(4 * var(--level-indent));
}
```

**Comment by farling42** [comment]
Implemented in 0.43.0
